test/wdio/slot-fallback-with-textnode/cmp.tsx

```diff
@@ -0,0 +1,21 @@
+import { Component, Fragment, h, State } from '@stencil/core';
+
+@Component({
+  tag: 'my-component',
+  shadow: false,
+  scoped: true,
+})
+export class MyComponent {
+  @State() shortName: string;
```

Comment:
Since we assign `null` in the `onClick` handler
```suggestion
  @State() shortName: null | string;
```

Reply:
done